crates/bevy_color/src/palettes/css.rs

```diff
@@ -0,0 +1,301 @@
+//! [Extended colors from the CSS4 specification](https://en.wikipedia.org/wiki/Web_colors#Extended_colors),
```

Comment:
IMO, we should link to the spec
https://www.w3.org/TR/css-color-4/#named-colors